<html><head><title>algal promoter analysis</title><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url('https://themes.googleusercontent.com/fonts/css?kit=wAPX1HepqA24RkYW1AuHYA');ol{margin:0;padding:0}.c2{max-width:468pt;background-color:#ffffff;padding:72pt 72pt 72pt 72pt}.c0{font-style:italic}.c1{direction:ltr}.title{padding-top:24pt;line-height:1.1500000000000001;text-align:left;color:#000000;font-size:36pt;font-family:"Calibri";font-weight:bold;padding-bottom:6pt}.subtitle{padding-top:18pt;line-height:1.1500000000000001;text-align:left;color:#666666;font-style:italic;font-size:24pt;font-family:"Georgia";padding-bottom:4pt}li{color:#000000;font-size:11pt;font-family:"Calibri"}p{color:#000000;font-size:11pt;margin:0;font-family:"Calibri"}h1{padding-top:24pt;line-height:1.1500000000000001;text-align:left;color:#000000;font-size:24pt;font-family:"Calibri";font-weight:bold;padding-bottom:6pt}h2{padding-top:18pt;line-height:1.1500000000000001;text-align:left;color:#000000;font-size:18pt;font-family:"Calibri";font-weight:bold;padding-bottom:4pt}h3{padding-top:14pt;line-height:1.1500000000000001;text-align:left;color:#000000;font-size:14pt;font-family:"Calibri";font-weight:bold;padding-bottom:4pt}h4{padding-top:12pt;line-height:1.1500000000000001;text-align:left;color:#000000;font-size:12pt;font-family:"Calibri";font-weight:bold;padding-bottom:2pt}h5{padding-top:11pt;line-height:1.1500000000000001;text-align:left;color:#000000;font-size:11pt;font-family:"Calibri";font-weight:bold;padding-bottom:2pt}h6{padding-top:10pt;line-height:1.1500000000000001;text-align:left;color:#000000;font-size:10pt;font-family:"Calibri";font-weight:bold;padding-bottom:2pt}</style></head><body class="c2"><p class="c1"><span>In the Mayfield lab, we are interested in understanding cis-motif logic and promoter design in green algae. &nbsp;We study this so that we can understand the unique aspects of promoters in algae, and equally as important - so that we can design and build recombinant gene expression systems in algae for the production of bio-products. &nbsp;Algae have enormous potential for the photosynthetic manufacturing of renewable products, like biofuels, nutraceuticals, and high value therapeutic proteins, but surprisingly, we lack sophisticated gene expression tools for algae, and we need to change that. We need to be able to control expression of both endogenous as well as exogenous proteins within algae. &nbsp;This control will allow us to to produce single high-value proteins or to engineer biosynthetic pathways in algae to produce specific biomolecules. While we have had some success using native promoters of highly expressed endogenous genes, for true industrial promise higher production and tighter control will be needed. </span></p><p class="c1"><a name="h.gjdgxs"></a><span>Currently, little is known about algal promoters (the DNA sequences that promote transcription of genes), but as eukaryotes they do share similarities with other well characterized organisms like yeast, mammalian cells, and plants. &nbsp;However, algal promoters and their promoter motifs have proven to be unique even in comparison to higher plants; for example promoters that drive high expression in plants do not functional in algae, and algal promoters for highly expressed genes seem to lack the ubiquitous &ldquo;TATA&rdquo; box found in almost all eukaryotic promoters. In many other organisms, understanding the specific DNA elements and the logic of how these motifs are arranged can inform the generation of synthetic promoters that can drive high and regulated transcriptions of genes of interest. &nbsp;</span></p><p class="c1"><span>In the Mayfield lab, we have previously collaborated with a small start-up company (GrassRoots) to take a bioinformatic approach to begin understanding promoter elements and logic in the model green algae, </span><span class="c0">Chlamydomonas</span><span>&nbsp;</span><span class="c0">reinhardtii. </span><span>&nbsp;Using motif searching tools (POWRs, Davis et al 2011) GrassRoots identified a number of DNA elements and their arrangement within the promoter regions of highly expressed </span><span class="c0">C. reinhardtii</span><span>&nbsp;genes. &nbsp;From this analysis they designed 25 different synthetic promoters, each 500 nucleotides in length, which contained these potential DNA elements in many different combinations. These synthetic promoters took into account both small sequence motifs (sets of short conserved sequences) as well as cis-motif logic (spacing, relative location, frequency and length of these small sequence motifs, etc.). </span></p><p class="c1"><span>In our lab, we tested the </span><span class="c0">in vivo</span><span>&nbsp;functionality of these synthetic promoters by placing them upstream of a fluorescent reporter gene that we subsequently transformed into algae. Promoter activity was then measured by fluorescence output from the transgenic algae, and many of these synthetic promoters were successful, although to varying degrees. &nbsp;The best synthetic promoters were only as good as the current best endogenous promoters that we normally use for recombinant gene expression. The plan now is to analyze the DNA motifs and cis-motif logic within both the &ldquo;good&rdquo; and &ldquo;bad&rdquo; synthetic promoters and correlate this to the relative expression of the reporter gene from each of the synthetic promoters. We then want to use this information to design a new set of synthetic promoters to identify if we can increase gene expression from the redesigned new set of synthetic promoters. </span></p><p class="c1"><span>Through this project we will be able to not only gain basic understanding of promoter structure in algae, but also produce a library of functional synthetic promoters that are essential for algal biotechnology to generate tailored algae for the production of bio-products and biofuels.</span></p></body></html>